public/js/script.js

```diff
@@ -0,0 +1,19 @@
+// JavaScript for disabling form submissions if there are invalid fields
+(() => {
+    'use strict'
+  
+    // Fetch all the forms we want to apply custom Bootstrap validation styles to
+    const forms = document.querySelectorAll('.needs-validation')
```

Comment:
Be more specific here on the selector to make the selected forms in the selector explicit as well. E.g. `form.needs-validation`.

Instead of the class you could use `data-*` attributes to indicate such forms. Why? Because class names should be used for styling purposes. You could add a data attribute to such forms and select them based on those.

E.g.
- Markup: `<form (...) data-needs-validation="true">(....)</form>`
- CSS selector: `form[data-needs-validation="true"]`